### PR TITLE
Move to net6

### PR DESCRIPTION
**Pull request by bannikovilea**
Hi, I noticed that the project references net 461. I needed to run it on net6. And so I moved it, I think it will be useful for others who use this scheduler.

I also try convert build script, but i got errors) So on net461 I got same)

### CODE REVIEW

**Comment by zidad** [comment]
Many thanks for updating! We should setup some automated builds and testss for this project as well :/

**Comment by bannikovilea** [comment]
I can set up github actions based on dotnet test. But I'm new to appveyor, gitversion and need help with packaging and publishing. 

Do you have a ci/cd workflow? :)

I set up tests in my [fork](https://github.com/bannikovilea/EasyNetQ.Scheduler/pull/1), but I had to exclude appveyor from the nuget configuration due to the fact that there are no necessary packages there or access token expired. :\

**Comment by bannikovilea** [comment]
So, if you replace 

```
<ClientLibraries Include="$(Source)\EasyNetQ\EasyNetQ.csproj" />
``` 

in Build/EasyNetQ.proj with 
```
<ClientLibraries Include="$(Source)\EasyNetQ.ExternalScheduler\EasyNetQ.ExternalScheduler.csproj" />
<ClientLibraries Include="$(Source)\EasyNetQ.Scheduler\EasyNetQ.Scheduler.csproj" />
<ClientLibraries Include="$(Source)\EasyNetQ.Scheduler.Mongo\EasyNetQ.Scheduler.Mongo.csproj" />
```

, then built.bat is executed successfully and will create the following files in Package folder:

- EasyNetQ.ExternalScheduler.2.3.0-use-git-versions.1.nupkg
- EasyNetQ.ExternalScheduler.2.3.0-use-git-versions.1.snupkg
- EasyNetQ.Scheduler.2.3.0-use-git-versions.1.nupkg
- EasyNetQ.Scheduler.Mongo.2.3.0-use-git-versions.1.nupkg

Is it right way? xD

**Comment by zidad** [comment]
Hey that's nice!

Though I would not use AppVeyor anymore, but just a simple build on github actions to start with. Also I don't think it necessarily has to publish packages, I think most people will use the sources of the scheduler in their project.

I would be fine with merging it as-is for now then add a build/tests later.

**Comment by bannikovilea** [comment]
Glad to hear the AppVeyor is no longer in use! 

So, I merge my github actions here)

It will run on the master branch and on pull requests to master

File: .github/workflows/dotnet-desktop.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+jobs:
+
+    build:
+
+        strategy:
+            matrix:
+                configuration: [Release]
+
+        runs-on: windows-latest  # For a list of available runner types, refer to
```
Comment (zidad) [inline .github/workflows/dotnet-desktop.yaml:55]: do we still need a windows image for anything?

File: .github/workflows/dotnet-desktop.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                  fetch-depth: 0
+
+            # Install the .NET Core workload
+            - name: Install .NET Core
+              uses: actions/setup-dotnet@v3
+              with:
+                  dotnet-version: 6.0.x
+
+            # Add  MSBuild to the PATH: https://github.com/microsoft/setup-msbuild
+            - name: Setup MSBuild.exe
```
Comment (zidad) [inline .github/workflows/dotnet-desktop.yaml:74]: is it needed if we only run dotnet?

**Review by zidad — APPROVED** [review]

File: .github/workflows/dotnet-desktop.yaml (hunk 1 above)
Comment (bannikovilea) [reply .github/workflows/dotnet-desktop.yaml:55]: It is needed because topshelf uses Windows system libraries to create services

Comment (bannikovilea) [reply .github/workflows/dotnet-desktop.yaml:55]: I'l try linux environment

**Pull request merged**